scripts/run_sc_server.py

```diff
@@ -0,0 +1,39 @@
+from os.path import join, abspath
+import os
+import argparse
+
+
+ostis_path = abspath(join(os.path.dirname(os.path.realpath(__file__)), "../"))
+
+
+def main(args: dict):
+    run_command = " ".join([join(ostis_path, 'bin/sc-server')])
+    if args['ext_path']:
+        path = abspath(join(os.getcwd(), args['ext_path']))
+        run_command = " ".join([run_command, '-e', path])
+    if args['bin_path']:
+        path = abspath(join(os.getcwd(), args['kb']))
+        run_command = " ".join([run_command, '--kb', path])
+    if args['config']:
+        path = abspath(join(os.getcwd(), args['config']))
+        run_command = " ".join([run_command, '--config', path])
+    if args['verbose']:
+        run_command = " ".join([run_command, '-v'])
+    if args['clear']:
+        run_command = " ".join([run_command, '--clear'])
```

Comment:
It can be simplified, it is necessary to make common calls

Reply:
Correct it, please

Reply:
Actually, I don't get your idea of simplification. However, in my opinion, this piece of code is pretty simple and easy to read and understand.